### DISCUSSION TITLE
How to list relevant directories?

### DISCUSSION THREAD

**Discussion by ghost** [ajeetdsouza/zoxide#367] [Q&A]
How can I do [something like this](https://github.com/ohmyzsh/ohmyzsh/blob/master/lib/directories.zsh) with zoxide to show the relevant directories instead of the latest ones?
```
function d () {
  if [[ -n $1 ]]; then
    dirs "$@"
  else
    dirs -v | head -n 10
  fi
}
compdef _dirs d
```

**Comment by ajeetdsouza** [comment]
How are you measuring relevance? zoxide's scoring algorithm factors in number of directory accesses as well as recency of access, so I'd say the result is both relevant and recent.

**Reply by ajeetdsouza** [reply]
Is `zoxide query -l` what you're looking for? You can add arguments to get just the directories matching your query.
